starter-code/src/components/button/Button.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import contacts from '../../../src/contacts.json';
+
+const Button = ({actors, actorName, name, isAdd, isName, isPop, isDel, updateState}) => {
+  const action = () => {
+    if (isAdd) {
+      const randomActor = contacts.splice(Math.floor(Math.random() * contacts.length), 1);
+      const newActors = actors.push(randomActor);
```

Comment:
`newActors` will be a `Number`, not an `Array`, because the **push** method returns the `Array.length`